integrations/gtag/lib/index.js

```diff
@@ -668,41 +667,47 @@ GTAG.prototype.timingCompletedEnhanced = function(track) {
  * @param  opt
  */
 
-function trackPageViewEvent(page, options) {
+GTAG.prototype.trackPageViewEvent = function(page, options) {
   var name = page.fullName();
   var category = page.category();
   var props = page.properties();
+  var campaign = page.proxy('context.campaign');
+  var track;
+
+  if (campaign && options.gaWebMeasurementId) {
+    push('config', options.gaWebMeasurementId, {
+      campaign: campaign
+    });
+  }
+
+  if (campaign && options.gaWebAppMeasurementId) {
+    push('config', options.gaWebAppMeasurementId, {
+      campaign: campaign
+    });
+  }
 
   var nonInteraction = !!(options && options.nonInteraction);
   var str = props.path;
   if (options.includeSearch && props.search) {
     str += props.search;
   }
-  if (options.trackAllPages) {
-    push('event', 'page_view', {
-      page_title: name || category,
-      page_location: props.url,
-      page_path: str,
-      non_interaction: nonInteraction
-    });
-  }
+
+  push('event', 'page_view', {
+    page_title: name || props.title,
```

Comment:
This was changed to stay inline with the existing GA integration: https://github.com/segmentio/analytics.js-integrations/blob/master/integrations/google-analytics/lib/index.js#L207